docker-compose.prod-dev.yml

```diff
@@ -0,0 +1,21 @@
+# Builds on docker-compose.yml
+# Runs the prod image in like dev mode, but as if it was deployed to staging.
+
+services:
+  dev-oidc:
+    ports:
+      - 3390:3390
```

Comment:
I don't think this is necessary since it should be coming from `docker-compose.dev.yml`.